config/crd/bases/configuration.konghq.com_kongingresses.yaml

```diff
@@ -130,7 +141,6 @@ spec:
                       probing.
                     properties:
                       concurrency:
-                        minimum: 1
```

Comment:
a minimum of one suggests that 0 is a problematic value here, and yet we're removing the validation that would provide early feedback about that. Is this something we can move into a build directive for kubebuilder so it can be added back?

Reply:
Everything under healthcheck is defined by `Healthchecks       *kong.Healthcheck json:"healthchecks,omitempty" yaml:"healthchecks,omitempty"`, so we source from the underlying go-kong type (which we do want, to avoid nonsense like the timeout/timeouts nonsense from earlier, and to get new fields automatically as they're added):

https://github.com/Kong/go-kong/blob/acbcf5e06e82d305ad1665cc778ed772eb6f09c9/kong/types.go#L134-L154

Historically we've shied away from adding anything Kubernetes-specific in there, though I'm personally fine with adding the annotations (they're comments, we probably shouldn't really care) there.

Reply:
[diff.txt](https://github.com/Kong/kubernetes-ingress-controller/files/7672252/diff.txt) shows the diff after https://github.com/Kong/go-kong/pull/104

Reply:
862db91f integrates the new version; same fix for all the remaining validation rules.